composer.json

```diff
@@ -96,10 +96,7 @@
         "static-code-analysis": "phpstan",
         "lint": "php-cs-fixer fix --dry-run",
         "fix": "php-cs-fixer fix",
-        "test": [
-            "php -r \"array_map('unlink', glob('./example/*/pacts/*.json'));\"",
-            "phpunit --debug"
-        ],
+        "clean-up-pacts": "php -r \"array_map('unlink', glob('./example/*/pacts/*.json'));\"",
```

Comment:
This will need to be documented as a breaking dependency. @YOU54F this change looks good; but how does PACT, or even PACT-PHP usually deal with this?

Also why would the process not be able to delete these files on windows? It feels like perhaps there is a locking bug?

Reply:
Reverted